File: C/Assignment7_2.c
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment7_2.c 
// Description : Write a program which accept doller from and return in indian currency.
// Author      : Aniket Devaba Markad 
// Date        : 16/05/2025
// 
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

#include<stdio.h>
#define INR 70

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 
// 
// Function Name : DollerToINR
// Description   : This function is used to calculate doller to indian currency. 
// Input         : int 
// Output        : int
// Author        : Aniket Devaba Markad 
// Date          : 16/05/2025 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

int DollerToINR(int iNo)
{  
    int iAmount = 0;

    if(iNo < 0)
    {
       iNo = -iNo;
    }  
  
    iAmount = iNo * INR ;
   
    return iAmount;
}

int main()
{
    int  iValue = 0;
    int  iRet = 0;
    
    printf("Enter amount in usd :");
    scanf("%d ",&iValue);

    iRet = DollerToINR(iValue);

    printf("Amount in indian currency :%d\n",iRet);

    return 0;
}
//Time Complexcity cant find in this example.

File: Java/Assignment7_2.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment7_2.java
// Description : write a program which Accept amount in us doller and return its corresponding value in Indian 
//               Currency consider 1 $ as 70 rupees
// input       : 10
// otput.      : 700
// input       : 1200
// otput.      : 84000
// Author      : Aniket Devaba Markad                        
// Date        : 1/1/2026
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class  Number
{

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name : DollerToInr
// Description   : This function is used to calculate doller in indian rupees
// Author        : Aniket Devaba Markad 
// Date          : 1/1/2026 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

    int DollerToInr(int iNo)
    {
        if(iNo < 0 )
        {
            System.out.println(" invalid input");
            return 0;
        }

        int iRuppes = 1;
        iRuppes = iNo * 70;

        return iRuppes;
    }
}
public class Assignment7_2
{
    public static void main(String a[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("enter the number ");
        int iValue = sobj.nextInt();

        Number nobj = new Number();
        int iRet = nobj.DollerToInr(iValue);

        System.out.println("amount is : "+ iRet);
    }
}

